lib/vagrant-openshift/command/test_openshift3_image.rb

```diff
@@ -97,9 +102,12 @@ def execute
 # get git sha1
 git_sha1=`git rev-parse --short #{ref}`
 
+[ "#{version}" != "" ] && cd #{version}
```

Comment:
+1 on the `-n`

I thought about the pushd & popd but wasnt sure about that, cause then it would be with one extra `if`.

[ -n "#{version}" ] && pushd #{version}
... some amazing code
[ -n "#{version}" ] && popd

but its tru it will be probably more readable. Will update. Thanks